email-ballerina/Module.md

```diff
@@ -0,0 +1,181 @@
+## Overview
```

Comment:
What if we have a high-level section for security which discusses about SSL, SmartTLS and default behavior. Then a bit on authentication.

Reply:
Also, I think we can explain a bit about the types of messages supported HTML and Text.

Reply:
Lastly we can explain a bit about attachments.

Reply:
Added the content.